<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EffectsPreviewPanel</class>
 <widget class="QWidget" name="EffectsPreviewPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Effects Preview</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string>font-weight: bold; background-color: #444; color: white; padding: 4px; font-size: 12px;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="previewContainer" native="true">
     <layout class="QVBoxLayout" name="previewContainerLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="viewControlsLayout">
     <item>
      <widget class="QPushButton" name="toggleViewButton">
       <property name="text">
        <string>2D/3D</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deviceOnlyButton">
       <property name="text">
        <string>Devices Only</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="changeLayerButton">
       <property name="text">
        <string>Change Layer</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetViewButton">
       <property name="text">
        <string>Reset View</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
